libzmix/bulletproofs_amcl/README.md

```diff
@@ -0,0 +1,5 @@
+# Bulletproofs
+
+1. On a pairing friendly curve. BLS12-381 and BN254 for now. Change value of default feature to change the curve,
+2. Using [Apache Milagro](https://github.com/milagro-crypto/amcl).
```

Comment:
Is there a reason to call out this dependency vs. others that are probably listed in cargo.toml?

Reply:
Because that dependency is the primary differentiator with dalek's implementation. I wanted to answer the question "Why another implementation when there is already one from Dalek."